topo/node/juniper/juniper.go

```diff
@@ -100,17 +100,21 @@ func (n *Node) SpawnCLIConn() error {
 
 // Returns config required to configure gRPC service
 func (n *Node) GRPCConfig() []string {
-	return []string{
+	port := n.Proto.Services[9339].GetInside()
```

Comment:
it would be better to iterate through the service names in the values of the map to find "gnmi" or any of the g* services that run on the port rather than assuming 9339

Reply:
9339 is standard gNMI but included lookup in the services map just to be safe.